### ISSUE TITLE
Emulator appears to be using ngrok for bots running locally

### ISSUE THREAD

**Issue by vishwacsena** [Microsoft/BotFramework-Emulator#974]
## Version
Latest V4

## Describe the bug
Talking to a locally running bot, I noticed that I'm running into ngrok limits. For bots running locally, I dont know why we need to use ngrok at all. 

```
Too many connections! The tunnel session &#39;abe726a3da7d3aa4583f16f944619a72&#39; has violated the rate-limit policy of 20 connections per minute by initiating 21 connections in the last 60 seconds. Please decrease your inbound connection volume or upgrade to a paid plan for additional capacity.
```
## To Reproduce
1. Start with any bot (Echo bot should suffice). Run it locally.
2. Try sending > 20 messages to the bot within a minute

## Expected behavior
There shouldnt be any rate limits hit for local development/ testing. 

[bug]

**Comment by justinwilaby** [comment]
I believe we should remove ngrok and allow the user the flexibility to choose and manage their own implementation. Making it a requirement to use the emulator for even a simple bot is like making all gloves with 6 fingers with the off chance someone with Polydactyly will want to buy them.

**Comment by vishwacsena** [comment]
Bringing this back into 4.1 release. This has caused confusion and issues with the recent user study as well. 

We need to (re)evaluate why we need to use ngrok to connect to bots running locally? If we really need this, then we need to update the first run UX as well as the settings UI to call out that this is a hard required.

**Comment by mewa1024** [comment]
Screenshot from the user study. User was using the most recent Mac version available morning of 10.19 (either Mac version 4.0.41930 or 4.0.41704.)

<img width="762" alt="ngrok-error" src="https://user-images.githubusercontent.com/39317542/47455098-d640e700-d785-11e8-97dd-2c909c1f7991.png">


And from 10.18 (Mac version 4.0.41704):

![screenshot 122](https://user-images.githubusercontent.com/39317542/47455264-494a5d80-d786-11e8-9231-bca9e58250b4.png)

A participant using the Windows version also was unable to connect from Emulator and remarked that he had a similar problem trying tools prior to the study and had to revert to using the v3 Emulator.

**Comment by justinwilaby** [comment]
It looks like this could be a regression. I'm seeing [code that conditionally uses the ngrok service URL](https://github.com/Microsoft/BotFramework-Emulator/blob/v4/packages/app/main/src/ngrokService.ts#L66) for requests. I'll investigate this.
